input/docs/integrations/editors/rider/run-configurations.md

```diff
@@ -0,0 +1,125 @@
+Order: 20
+Title: Run configurations
+Description: Run configurations for Cake tasks
+---
+
+In order to use the run configurations you must have Cake installed on your machine or in your solution.
```

Comment:
I would suggest to use the same heading structure here as we have used for [Visual Studio](http://cakebuild.net/docs/integrations/editors/visualstudio/task-runner):

- `Configuring run configurations`
  - `Install Cake`
  - `Supported runners`
  - `Settings`  (instead of `Settings regarding the run configurations`)
- `Using run configurations`
  - `Creating run configurations`
  - `Editing run configurations`
  - `Running builds` (instead of `Cake tasks tool window`)